SPECS/libacvp/libacvp.spec

```
@@ -35,7 +35,7 @@ and the library itself.
 ./configure \
     --prefix=%{_prefix} \
     --enable-offline \
-    CFLAGS="-pthread -DACVP_NO_RUNTIME -DOPENSSL_KWP -DOPENSSL_KDF_SUPPORT -O0 -g" \
+    CFLAGS="$CFLAGS -pthread" \
```

Revert this line. The original CFLAGS is important to preserve because it enables specific OpenSSL vectors needed for libacvp testing.

---

But I wouldn't want to lose our global `CFLAGS` values. Would something like this be acceptable instead:

```suggestion
    CFLAGS="$CFLAGS -pthread -DACVP_NO_RUNTIME -DOPENSSL_KWP -DOPENSSL_KDF_SUPPORT -O0 -g" \
```

---

We would need to confirm whether the resulting binary still does what we expect it to do with the $CFLAGS added in. It would likely be fine but without confirming, I am of the opinion that it is best to keep the original CFLAGS override until we can confirm adding $CFLAGS is truly safe.

---

I've reverted the change here